frontend/src/components/Skill/SkillGroup/SkillsGroup.tsx

```diff
@@ -0,0 +1,31 @@
+import { FC, Suspense } from 'react';
+
+import { LENGTH } from 'constants/sizes';
+import Loader from 'components/Loader';
+
+import { SkillsTitle, SkillsBox, SkillsDivider } from './styled';
+import SkillItem from '../SkillItem/SkillItem';
+import { ISkillsGroupProps } from '../types';
+
+const SkillsGroup: FC<ISkillsGroupProps> = ({ skillFounded, skills }) => {
+  const skillsData = skillFounded.length ? skillFounded : skills;
+  return (
+    <>
+      {skillsData?.map((skillGroup, key) => {
+        const isDividerVisible = key < skillsData.length + LENGTH.lastArrayItem;
+        return (
+          <Suspense key={skillGroup.name} fallback={<Loader color="primary" type="component" />}>
```

Comment:
We should use Suspense here? https://www.youtube.com/watch?v=IU6WpNSSNOI&ab_channel=%D0%92%D0%BB%D0%B0%D0%B4%D0%B8%D0%BB%D0%B5%D0%BD%D0%9C%D0%B8%D0%BD%D0%B8%D0%BD. Or https://www.youtube.com/watch?v=NTDJ-NQ32_E&ab_channel=TraversyMedia